src/main/java/com/monumental/models/Contribution.java

```diff
@@ -18,6 +19,7 @@
 public class Contribution extends Model implements Serializable {
 
     @Column(name = "submitted_by")
+    @NotNull(groups = NewOrExisting.class, message = "Submitted By can not be null")
```

Comment:
`@NotNull` is one of the JPA constraint annotations and allows for a custom message when violated.